ctapipe/io/containers.py

```diff
@@ -78,6 +77,7 @@ class RawCameraContainer(Container):
                             "integrated ADC data (n_pixels)"))
     adc_samples = Item(Map(), ("map of channel to arrays of "
                                "(n_pixels, n_samples)"))
+    num_samples = Item(int, "number of time samples")
```

Comment:
I very much dislike the way it is right now. But there does not seem to be an alternate method in hessio to obtain number of samples directly. Perhaps the best option would be to not store this value, forcing the user to obtain the number of samples from adc_samples.shape.